### PR TITLE
enterprise update

### PR DESCRIPTION
**Pull request by sinzone**
@thibaultCha pls review, merge and deploy.

### CODE REVIEW

File: app/enterprise/index.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
   <div class="container">
     <header class="section-header">
       <h2>Move faster with Enterprise grade features.</h2>
-      <p>Kong offers subscription plans for Priority Support. Products, Consulting &amp; Training optional.</p>
+      <p>Kong offers subscription plans for access to Priority Support, Kong Manager, Consulting &amp; Training.</p>
```
Comment (ahmadnassri) [inline app/enterprise/index.html]: "for access" should be "with access"

File: app/enterprise/index.html [hunk 2]
```diff
@@ -43,37 +43,38 @@
...
+            <h4>Kong Enterprise Subscription</h4>
           </li>
           <li class="plan-line muted">
 
-            <h5>Official Production Support</h5>
+            <h5>Products</h5>
+          </li>
+          <li class="plan-line"><a href="http://www.getgalileo.io" target="_blank">Galileo</a> - API Analytics</li>
+          <li class="plan-line"><a href="http://www.gelato.io" target="_blank">Gelato Docs</a> - Dev Portal/Hub</li>
+          <li class="plan-line">Jungle - Kong Admin Manager (Q3 2016)</li>
```
Comment (ahmadnassri) [inline app/enterprise/index.html]: don't use "Jungle", already used "Kong Manager" above, stick to that

File: app/enterprise/index.html [hunk 3]
```diff
@@ -123,7 +124,7 @@
         <label for="phone">Phone <small>(optional)</small></label>
         <input id="phone" name="phone" type="text" placeholder="415 000 0000">
 
-        <label for="tell-us-more">Tell us more <small>(optional)</small></label>
+        <label for="tell-us-more">Tell us more></label>
```
Comment (ahmadnassri) [inline app/enterprise/index.html]: extra `>`

Comment (ahmadnassri) [inline app/enterprise/index.html]: @theetrain plz fix, merge and publish

File: app/enterprise/index.html [hunk 4]
```diff
@@ -28,8 +28,8 @@
 <section class="section plans-section">
   <div class="container">
     <header class="section-header">
-      <h2>Move faster with Enterprise grade features.</h2>
-      <p>Kong offers subscription plans for Priority Support. Products, Consulting &amp; Training optional.</p>
+      <h2>Move faster with Enterprise-grade features.</h2>
```
Comment (theetrain) [inline app/enterprise/index.html]: Any thoughts on this change?

File: app/enterprise/index.html [hunk 5]
```diff
@@ -43,37 +43,38 @@
...
+          <li class="plan-line muted">
+             <h5>Official Production Support</h5>
           </li>
           <li class="plan-line">Email, Chat &amp; Phone Support with Kong Engineers</li>
-          <li class="plan-line">1 to 24 Hours Response Times</li>
-          <li class="plan-line">10 to Unlimited Max Annual Incidents</li>
-          <li class="plan-line">Initial Set-Up</li>
+          <li class="plan-line">30 Mins to 8 Hours Response Times</li>
+          <li class="plan-line">25 to Unlimited Max Annual Incidents</li>
+          <li class="plan-line">Initial Setup</li>
```
Comment (theetrain) [inline app/enterprise/index.html]: Any thoughts?

File: app/enterprise/index.html [hunk 6]
```diff
@@ -123,7 +124,7 @@
         <label for="phone">Phone <small>(optional)</small></label>
         <input id="phone" name="phone" type="text" placeholder="415 000 0000">
 
-        <label for="tell-us-more">Tell us more <small>(optional)</small></label>
+        <label for="tell-us-more">Tell us more</label>
```
Comment (theetrain) [inline app/enterprise/index.html]: Removed angled bracket

**Comment by theetrain** [comment]
Added all my changes.

File: app/enterprise/index.html (hunk 4 above)
Comment (ahmadnassri) [inline app/enterprise/index.html]: :-1:

File: app/enterprise/index.html (hunk 5 above)
Comment (ahmadnassri) [inline app/enterprise/index.html]: :+1:

**Pull request merged**